spec/spec_helper.rb

```diff
@@ -52,7 +52,12 @@
 
   if GoodData::Environment::VCR_ON
     require 'vcr_configurer'
-    skip_sleep = GoodData::Helpers::VcrConfigurer.vcr_record_mode == :none
+    if GoodData::Helpers::VcrConfigurer.vcr_record_mode == :none
+      skip_sleep = true
+      GoodData::Helpers::ProjectHelper.project_id = 'VCRFakeId'
```

Comment:
The `VcrFake*` strings are used multiple times, could it be somehow extracted/shared to/from one place?